Source/WebKit/WebProcess/WebProcess.cpp

```diff
@@ -2335,6 +2339,12 @@ void WebProcess::removeWebTransportSession(WebTransportSessionIdentifier identif
     m_webTransportSessions.remove(identifier);
 }
 
+void WebProcess::updateCachedCookiesEnabled()
+{
+    for (auto& document : Document::allDocuments())
```

Comment:
`Ref`

Reply:
It turns out allDocuments()  returns CheckRef, and it gives compile error when using Ref here; probably because CheckedRef cannot be converted to Ref directly.